app/rtc_s1.py

```diff
@@ -326,12 +334,14 @@ def calculate_layover_shadow_mask(burst_in: Sentinel1BurstSlc,
                                       geogrid_in.width, geogrid_in.length, 1,
                                       gdal.GDT_Byte, output_raster_format)
 
-    geo.geocode(radar_grid=rdr_grid,
-                input_raster=mask_raster,
+    geo.geocode(radar_grid=radar_grid,
+                input_raster=layover_shadow_mask_raster,
                 output_raster=geocoded_raster,
                 dem_raster=dem_raster,
                 output_mode=isce3.geocode.GeocodeOutputMode.INTERP)
 
+    return layover_shadow_mask_raster
```

Comment:
I know the variable name is already long, but how about we add something like `radar_grid` to clarity that this mask is in radar grid?

Reply:
Yes, good idea. Thanks